src/main/java/org/gridsuite/securityanalysis/server/CassandraConfig.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.securityanalysis.server;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+import org.springframework.data.cassandra.config.AbstractCassandraConfiguration;
+import org.springframework.data.cassandra.config.CassandraClusterFactoryBean;
+import org.springframework.data.cassandra.repository.config.EnableCassandraRepositories;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+@Configuration
+@PropertySource(value = {"classpath:cassandra.properties"})
+@PropertySource(value = {"file:/config/cassandra.properties"}, ignoreResourceNotFound = true)
+@EnableCassandraRepositories(basePackageClasses = {LimitViolationRepository.class})
```

Comment:
Maybe we can list all the repository classes here, separated with ','

Reply:
We can remove all basePackageClasses as default behaviour is to scan current package and its children